File: C_Programming/Practise_Programs/PP_042.c
//////////////////////////////////////////////////////////////////
//
//  File Name         : PP_042
//
//  Description       : This program accepts a number from the user
//                      and displays negative numbers starting from
//                      the negative of the given number up to -1.
//
//  Author            : Shubham Somanath Gadhe
//  Date              : 08/07/2026
//
//  Time Complexity   : O(n)
//  Space Complexity  : O(1)
//
//////////////////////////////////////////////////////////////////

#include<stdio.h>

//////////////////////////////////////////////////////////////////
//
//  Function Name     : Display()
//
//  Description       : It is used to display negative numbers
//                      starting from the negative of the given
//                      number up to -1 using a for loop.
//
//  Author            : Shubham Somanath Gadhe
//  Date              : 08/07/2026
//
//////////////////////////////////////////////////////////////////

void Display(int iNo)
{
    int iCnt = 0;

    for(iCnt = -iNo; iCnt < 0; iCnt++)
    {
        printf("%d\t", iCnt);
    }

    printf("\n");
}

int main()
{
    int iValue = 0;

    printf("Enter number : \n");
    scanf("%d", &iValue);

    Display(iValue);

    return 0;
}

//////////////////////////////////////////////////////////////////
//
//  Sample Input  :
//      Enter number :
//      5
//
//  Sample Output :
//      -5    -4    -3    -2    -1
//
//////////////////////////////////////////////////////////////////

File: Java_Programming/Practise_Programs/PP_042.java
//////////////////////////////////////////////////////////////////
//
//  File Name         : PP_042
//
//  Description       : This program accepts a number from the user
//                      and displays each number enclosed
//                      between '*' symbols.
//
//  Author            : Shubham Somanath Gadhe
//  Date              : 06/08/2026
//
//  Time Complexity   : O(n)
//  Space Complexity  : O(1)
//
//////////////////////////////////////////////////////////////////

import java.util.*;

class PP_042
{
    //////////////////////////////////////////////////////////////////
    //
    //  Function Name    : Display()
    //
    //  Description      : It is used to display numbers
    //                     from 1 to the specified number,
    //                     with each number enclosed
    //                     between '*' symbols.
    //
    //  Author           : Shubham Somanath Gadhe
    //  Date             : 06/08/2026
    //
    //////////////////////////////////////////////////////////////////

    public static void Display(int iNo)
    {
        int iCnt = 0;

        for(iCnt = 1; iCnt <= iNo; iCnt++)
        {
            System.out.print("*" + iCnt + "*\t");
        }

        System.out.println();
    }

    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        int iValue = 0;

        System.out.println("Enter the number of elements : ");
        iValue = sobj.nextInt();

        Display(iValue);
    }
}

//////////////////////////////////////////////////////////////////
//
//  Sample Input  :
//      5
//
//  Sample Output :
//      *1*    *2*    *3*    *4*    *5*
//
//////////////////////////////////////////////////////////////////